kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UserTaskServicesClientImpl.java

```diff
@@ -132,6 +132,20 @@ public void claimTask(String containerId, Long taskId, String userId) {
             throwExceptionOnFailure(response);
         }
     }
+    
+    @Override
+    public void claimTask(String containerId, List<Long> taskIds, String userId) {
+        if( config.isRest() ) {
+            String queryString = buildQueryString("taskId",taskIds);
+            queryString+= getUserQueryStr(userId);
+            makeHttpPutRequestAndCreateCustomResponse(
+                    build(loadBalancer.getUrl(), TASK_URI + "/" + TASK_INSTANCE_CLAIM_PUT_URI, Collections.singletonMap(CONTAINER_ID, containerId)) + queryString, null, String.class, getHeaders(null));
```

Comment:
Shouldn't it be TASK_INSTANCE_CLAIM_BULK_URI?
```suggestion
                    build(loadBalancer.getUrl(), TASK_URI + "/" + TASK_INSTANCE_CLAIM_BULK_URI, Collections.singletonMap(CONTAINER_ID, containerId)) + queryString, null, String.class, getHeaders(null));
```

Reply:
Thanks!!!